cli/command/formatter/cluster_health_test.go

```diff
@@ -0,0 +1,109 @@
+package formatter
+
+import (
+	"bytes"
+	"testing"
+
+	apiTypes "github.com/storageos/go-api/types"
+	"github.com/storageos/go-cli/types"
+)
+
+func TestClusterHealthContextWrite(t *testing.T) {
```

Comment:
Let's rename this to `TestClusterHealthWrite` as that's the function being called here, which then calls and indirectly tests other functions as well.